File: solucoes/2020/nivel 1/fase 1/py/emoticons.py
M = input()
d = 0
c = 0

for i in range(len(M)):
    if M[i] == '(' and M[i-1] == "-" and M[i-2] == ":":
        c += 1
    elif M[i] == ')' and M[i-1] == "-" and M[i-2] == ":":
        d += 1

if d > c:
    print("divertido")
elif c > d:
    print("chateado")
else:
    print("neutro")

File: solucoes/2020/nivel 1/fase 1/js/emoticons.js
var C = 0
var D = 0
var c, msg;

scanf("%c","c");
while (c != '\n') {
	msg += c;
	scanf("%c","c");
}

for(var i = 0; i < msg.length; i++){
    if(M[i] == ")" ){
        D++
    }
    else if(M[i] == "("){
        C++
    }
}

if(C > D){
    printf("%s", "chateado")
}
else if(D > C){
    printf("%s", "divertido")
}
else{
    printf("%s", "neutro")
}